docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
+
+Linking internal `Group`s of users to their representations on LDAP servers allows for administrators to have an easier way to manage groups inside OpenShift. A command like `oadm sync groups` or `oadm sync group <name>` will trigger a sync of group members with their external support (if any).
```

Comment:
Learn how identities map to users and figure out what information will be required to properly sync a User into a group.  I expect that there will be a one to one mapping between ldap user and openshift identity and an openshift identity may have an openshift user and the openshift user is what needs to added to any openshift group.